ccd-definition/ComplexTypes/CareSupervision/Orders.json

```diff
@@ -107,7 +107,9 @@
     "ListElementCode": "directionLabel",
     "FieldType": "Label",
     "ElementLabel": "## Directions",
-    "SecurityClassification": "Public"
+    "SecurityClassification": "Public",
+    "FieldShowCondition": "directionLabel=\"HIDE_LABEL\"",
+    "Comment": "Needs showing in event, but requires configuring a CaseEventToComplexType for ordersNeeded"
```

Comment:
this will remove the header from the event, but will hide it from the tab - thoughts? is it that important in event?

Reply:
I would say we should maintain old functionality so even though it is more work it might be worth while making a CEtCT file for it

Reply:
yea will revert for now and handle separately